### ISSUE TITLE
Meson devenv fails with the MSI installed version

### ISSUE THREAD

**Issue by elmarco** [mesonbuild/meson#9872]
meson --version 0.61.1, installed from MSI on Win11.

```
> meson devenv
Traceback (most recent call last):
  File "mesonbuild\mesonmain.py", line 146, in run
  File "mesonbuild\mdevenv.py", line 52, in run
  File "mesonbuild\mdevenv.py", line 21, in get_windows_shell
  File "subprocess.py", line 420, in check_output
  File "subprocess.py", line 524, in run
subprocess.CalledProcessError: Command '['powershell.exe', '-noprofile', '-executionpolicy', 'bypass', '-file', 'C:\\Program Files\\Meson\\mesonbuild\\scripts\\cmd_or_ps.ps1']' returned non-zero exit status 4294770688.

ERROR: Unhandled python exception

    This is a Meson bug and should be reported!
```

`msiextract -l ~/Downloads/meson-0.61.1-64.msi` shows the file is missing

**Comment by xclaesse** [comment]
Does `C:\\Program Files\\Meson\\mesonbuild\\scripts\\cmd_or_ps.ps1` exists for you?

**Comment by eli-schwartz** [comment]
If the file doesn't exist in the msi archive, I can only presume it must not exist once extracted either...

**Comment by eli-schwartz** [comment]
@jpakkane seems like for some reason `packaging/createmsi.py` specifies a lot of python modules manually, but it's not clear whether it includes non-python data files.

**Comment by xclaesse** [comment]
Oops, haven't read the bug report to the end. Sorry for the noise ahah. :D

**Comment by punytroll** [comment]
Any updates on this? This still fails for Meson 0.62.0 installed from MSI.

I can confirm that, copying the cmd_or_ps.ps1 file from github to the mesonbuild/scripts folder resolves this issue and the "meson devenv" command works. (It still fails when calling an executable through it, but that seems unrelated.) The script file is never mentioned in createmsi.py. It doesn't seem reasonable to create mesonbuild/scripts subdirectories solely for the missing ps1 script, but that seems hardcoded in mdevenv.py.

**Comment by eli-schwartz** [comment]
Sorry, this seems to be the exact same problem later reported by #10163, and both should be fixed by #10230

This will work again for the 0.62.1 MSI once the new point release is out.
